docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
+
+This guide explains how to restrict privileges for deploying Scalar DL on Azure.
+
+In general from security perspective, it is better to assign restricted privileges to users not to make them able to do operations that they are not supposed to do.
```

Comment:
```suggestion
 From the security perspective, It is better to give users limited privileges to protect the environment from unwanted activities.
```

Reply:
You can change the suggestion in your own way.

Reply:
Thank you for the suggestion, updated sentence in c601da6